### ISSUE TITLE
ARIA role must be appropriate for the element

### ISSUE THREAD

**Issue by stef-barburou** [snap-framework/snap#19]
Language: English

Tool(s): aXe

Browser/Screen reader: Firefox

WCAG 2.0 reference: Deque best practice

Details: aXe reported that : ARIA role must be appropriate for the element
Issue description: Ensures role attribute has an appropriate value for the element
Why it matters: Intended accessible technology behavior by a developer is not enabled when an assigned WAI-ARIA role value is invalid for the parent element.

When screen readers and other assistive technologies do not know the role of each element on the web page, they are not able to interact with it intelligently, nor are they able to communicate the role to the user. When the value for a role is invalid, there is no way to communicate the element's features, properties, and methods to assistive technologies. For example, applying role="table" to a <ul> effectively hijacks the default semantics associated with the <ul> element in a way that screenreaders do not expect resulting in unexpected behavior.

Element location:
#wb-tphp
Element location:
<ul id="wb-tphp" class="wb-init wb-disable-inited" role="navigation"> <li class="wb-slc"> <a class="wb-sl" href="#dynamic_content">Skip to main content</a> </li> </ul>

To solve this violation, you need to:
Fix the following: ARIA role navigation is not allowed for given element
